collect_app/src/main/java/org/odk/collect/android/dao/FormsDao.java

```diff
@@ -72,6 +72,17 @@ public Cursor getFormsCursor(String[] projection, String selection, String[] sel
     public void deleteFormsDatabase() {
         Collect.getInstance().getContentResolver().delete(FormsProviderAPI.FormsColumns.CONTENT_URI, null, null);
     }
+    
+    public void deleteFormsFromIDs(String[] idsToDelete) {
+
```

Comment:
We try to follow [the Android contributor style guide](https://source.android.com/source/code-style.html) which does not include blank lines at the beginning and end of methods. Please remove those!